helm/fiftyone-teams-app/README.md

```diff
@@ -93,13 +93,12 @@ documentation before completing your upgrade.
 
 Voxel51 recommends upgrading your deployment using
 [`legacy` authentication mode](https://docs.voxel51.com/teams/pluggable_auth.html#legacy-mode)
-and migrating to
```

Comment:
Let's cherry pick the PR that fixes this file...

Reply:
you sure are asking for a lot of changes to a release that's not supposed to have many changes...

I'll look into it.

Reply:
That commit has 11 files changed in it:
https://github.com/voxel51/fiftyone-teams-app-deploy/pull/137/commits/85f0a0ffa7c870871023407dd64e7e4a25a9322f

I think I have manually moved over the changes you're asking for.